pkgs/by-name/ka/kaldi/package.nix

```diff
@@ -34,6 +34,8 @@ stdenv.mkDerivation (finalAttrs: {
     "-DBLAS_LIBRARIES=-lblas"
     "-DLAPACK_LIBRARIES=-llapack"
     "-DFETCHCONTENT_SOURCE_DIR_OPENFST:PATH=${finalAttrs.passthru.sources.openfst}"
+    # Fix the build with CMake 4.
+    "-DCMAKE_POLICY_VERSION_MINIMUM=3.10"
```

Comment:
Maybe https://github.com/kkm000/openfst/commit/f75b38207293ed6a3aab97e1389de6727f584acd could be patch on `passthru.sources` instead?